pkgs/development/python-modules/attrs-strict/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  python3
+}:
+
+buildPythonPackage rec {
+  pname = "attrs_strict";
+  version = "1.0.1";
+  disabled = pythonOlder "3.7";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-5wSGNiAUbF8qi2Ac1FdNFIkT2yb8Bjb5Qf5CEuQl6v4=";
+  };
+
+  build-system = [
+    python3.pkgs.setuptools
+    python3.pkgs.setuptools-scm
+  ];
+
+  dependencies = with python3.pkgs; [
+    attrs
+  ];
```

Comment:
Consume these directly as function args when packaging within python-modules.

The way you are currently packaging them makes them stick to the current `python3` and prevent multiple versioned package sets.